pkgs/tools/wayland/waynergy/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub
+, meson, pkg-config, ninja, wayland-scanner
+, libdrm, wayland, wayland-protocols
+, wl-clipboard
+, libxkbcommon, cmake
+, libressl
+}:
+stdenv.mkDerivation rec {
+  pname = "waynergy";
+  version = "0.0.13";
+
+  src = fetchFromGitHub {
+    owner = "r-c-f";
+    repo = "waynergy";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-eTY7tktUmoTZO3w9uP1P8cIz0mmFiWm5YFGVAS6JwwE=";
```

You either do (preferred):
```suggestion
    hash = "sha256-eTY7tktUmoTZO3w9uP1P8cIz0mmFiWm5YFGVAS6JwwE=";
```
Or use a base16/base32/base64 **unprefixed** sha256 with `sha256 = `.

See more at: https://nixos.wiki/wiki/Nix_Hash#Usage